packages/sui-ssr/server/criticalCss/index.js

```diff
@@ -63,115 +75,114 @@ export default config => (req, res, next) => {
     mobile: 'm'
   }
   const device = deviceTypes[type] || deviceTypes.desktop
-  const {url} = req
 
-  match(
-    {routes, location: url},
-    async (error, redirectLocation, renderProps) => {
-      if (error) {
-        return next(error)
-      }
+  if (
+    Array.isArray(currentConfig.blackListRoutePaths) &&
+    currentConfig.blackListRoutePaths.some(routePath =>
+      renderProps.routes.some(route => route.path === routePath)
+    )
+  ) {
+    logMessage('Skip middleware because route path is blacklisted')
+    return next()
+  }
 
-      if (!renderProps) {
-        return next()
-      }
+  const hash = generateMinimalCSSHash(renderProps.routes) + '|' + device
+  const criticalCSS = __CACHE__[hash]
+  const retrysByHash = __RETRYS_BY_HASH__[hash] || 0
```

Comment:
plural of retry is retries

Reply:
This code was already implemented, it's marked as modified because of changes in indentation... although I can make a linting PR later to follow the boy scout principle :)